Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -450,8 +450,8 @@
   // See http://minow.blogspot.com/index.html#4918805519571907051
   #define DELTA_CALIBRATION_MENU
 
-  // set the radius for the calibration probe points - max 0.8 * DELTA_PRINTABLE_RADIUS if DELTA_AUTO_CALIBRATION enabled
-  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS - 17) // mm
+  // set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS*0.869 if DELTA_AUTO_CALIBRATION enabled
+  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS*0.869) // mm
```

Comment:
All preprocessor defines must be parenthesized if used like so:
```cpp
(DEFINED_THING) * 2.0
(DEFINED_THING) / 2.0
SOMETHING - (DEFINED_THING)
```
Basically, imagine that `DEFINED_THING` is equal to the string "3 + 10", and then imagine what would happen when `7 * DEFINED_THING` is expanded. Clearly, the result `7 * 3 + 10` is bad, whereas `7 * (3 + 10)` is correct.

Where are parentheses not needed? Usually, this is fine and we let it slide:
```cpp
17 + DEFINED_THING
```
Since that won't be a problem if `DEFINED_THING` is "3 + 10". However, we may tighten up on that as well, since if `DEFINED_THING` is equal to something like "flag ? 5 : 10" even the expansion of a simple addition will be incorrect.
```cpp
17 + flag ? 5 : 10
```